src/main/webapp/app/exercises/text/assess/text-submission-assessment.route.ts

```diff
@@ -101,7 +101,7 @@ export const textSubmissionAssessmentRoutes: Routes = [
         canActivate: [UserRouteAccessService],
     },
     {
-        path: 'submissions/:submissionId/assessment',
+        path: 'participations/:participationId/submissions/:submissionId/for-text-assessment',
```

Comment:
if you change those paths, you probably also need to change the component that load the corresponding pages